File: 1º Semestre/Introd Comp e Tec/aula3/ex2.py
# Desenvolva um programa em Python que solicite ao usuário dois intervalos [a,b] e [c,d] e verifique se esses intervalos se interceptam.
# Exemplo:
# Intervalos [0, 1] e [1, 2] se interceptam.
# Intervalos [1, 2] e [0, 1] se interceptam.
# Intervalos [0, 1] e [2, 3] não se interceptam.
# Intervalos [2, 3] e [0, 1] não se interceptam.
# Intervalos [0, 3] e [1, 2] se interceptam.

a = int(input("Digite a: "))
b = int(input("Digite b: "))
c = int(input("Digite c: "))
d = int(input("Digite d: "))

if a > b or c > d:
    print("Dados Inválidos")
elif b < c or d < a:
    print("Não se interceptam")
else:
    print("Se Interceptam")

File: 1º Semestre/Programacao de Computadores 1/aula2/ex2.c
// Desenvolva um programa que solicite ao usuário digitar dois valores inteiros. 
// Após a digitação, o computador deverá executar a subtração do primeiro número do segundo número e mostrar o resultado na tela do computador.
// Exemplo: se o usuário digitar os valores 900 e 100, o programa deve realizar a subtração do número 900 pelo número 100 e mostrar na tela o resultado (800).

#include <stdio.h>

int main () {
  int n1,n2;

  printf("Digite o primeiro numero: ");
  scanf("%d", &n1);
  printf("Digite o segundo numero: ");
  scanf("%d", &n2);

  printf("%d",n1-n2);

  return 0;
}
